data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,470 @@ mission "Unfettered returning home"
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Anomalocaris, or some of the cargo was on another ship.`
+	to complete
+		has "ship model: Anomalocaris"
+	on complete
+		dialog `After landing, a group of Unfettered gather around your ship and begin unloading the medical supplies, though strangely they don't seem to be in any hurry. Shortly after they've finished, you receive a rather brief message from the Unfettered you bought the ship from. "Thank you for helping our brethren, <last>. Please accept these <payment>."`
+		payment 1500000
+		log "Delivered a pressing food shipment to the Unfettered Hai using an Anomalocaris, which seems to have pleased them."
+
+
+
+mission "Hai Legacy 3"
+	landing
+	name "Ancient Claim"
+	description "Beat the Lightning Bugs in combat. You only have one day, and becoming disabled will result in losing the fight."
+	source "Firelode"
+	deadline 2
+	to offer
+		has "Hai Legacy 2: done"
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`As you land, you realize your ship is the center of attention. One Unfettered that looks like an engineer comes to speak to you, warmer than usual compared to your previous experiences with the Unfettered.`
+			`	"Welcome. Although I do not know how you have obtained this ship, I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of. As such, most Unfettered warriors will say that you need to prove your right to use that ship to us. Luckily for you, I don't believe in any of that. What matters to me is that this ship is very durable, and it's filled with valuable ancient technology.`
+			`	"I even see technology from those savage Korath that we fought valiantly to obtain!"`
+				to display
+					has "outfit: Korath Detainer"
+			`	Her tone suddenly changes as she raises her voice for everyone to hear. "I, Mera, challenge you! The winner will earn the right to own this honorable relic and its mighty weapons!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+				`	"I bought this ship. It's mine and I shall do what I please with it. I'm leaving now."`
+					goto headingout
+			`	"No, because in the eyes of my brethren that would mean I, too, did not deserve this ship. They would then challenge me for it."`
+			label win
+			`	"You will win the right to access our worlds. In order to keep our respect, this Anomalocaris may not be destroyed or sold, until you prove yourself another way. This is a priceless artifact. Should you refuse to prove your worth, you will no longer be allowed to land here without paying a fee.`
+			`	"Beware, for there are no official rules, other than the fact that you may only use this one ship. So, will you take to the skies, or are you too afraid?"`
```

Comment:
```suggestion
			`	Mera turns around, talking to other Unfettered for a bit before addressing you again. "You will win the right to access our worlds. But, in order to keep our respect, you must keep this Anomalocaris in good condition until you prove yourself another way. It is a priceless artifact. Should you refuse to prove your worth, you will no longer be allowed to land here.`
			`	She then glares at you. "Beware, for although there are no official rules, you must use only the Anomalocaris. So, will you take to the skies, or are you too afraid?"`
```
A few more additions and changes narratively. It's a little odd she all of a sudden is speaking for all the Unfettered without pause about what winning means, as if she's not just an engineer but a leader of all of them, so I've added a little intro to that. Saying don't sell or destroy the ship is a bit too on the nose, I think it's fine we just say, "keep it in good condition" meaning the same thing. And again, more breaking up quotes. We try to avoid having multiple lines if the conversation is short, but if we do so, it's good to add a bit more narrative element to it where feasible.

Reply:
she isn't *really* a leader but she makes prototypes weapons they all want to try, giving her influence
but the changes sound good